packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -56,20 +56,36 @@ export default class TelecomPackMigrationOffersCtrl {
 
   updateOfferDisplayedPrice(value, offer, optionName) {
     let totalOfferPrice = offer.price.value;
+    let optionComfort = 0;
```

Comment:
Why using numeric value here, while it is used as a boolean?

Reply:
change it to boolean